src/io/flutter/utils/MostlySilentOsProcessHandler.java

```diff
@@ -30,14 +32,33 @@
  * for more information.
  */
 public class MostlySilentOsProcessHandler extends OSProcessHandler {
+  private boolean softKill;
```

Comment:
Add a comment indicating why softKill should be preferred. 
Your comment for this CL is about the right level of granularity.
`SIGKILL is currently sent when a test process is stopped, but this can't be routed to remote processes spawned from the original one.`